database/database.go

```diff
@@ -140,6 +142,36 @@ func (d *Database) GetUser(username string) (user model.User, err error) {
 	return user, nil
 }
 
+// GetUsers will return a list of users whose username matches a search string.
+func (d *Database) GetUsers(search string) ([]model.User, error) {
+	users := []model.User{}
+	sqlStatement := `
+		SELECT Id, Username
+		FROM (SELECT Id, to_tsvector(Username) as lex, UserRole, Username
```

Comment:
Really cool that you used this! Does it do fuzzy stuff?

Reply:
I don't think this one does fuzzy stuff, but just noticed that article has some info on fuzzy search, I'll take a look at that.